portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,11 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+function goToHomePage() {
```

Comment:
All the Javascript functions should have JSDoc https://google.github.io/styleguide/jsguide.html#jsdoc-general-form